tests/test_lock.py

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+from drgn_tools import lock
+
+
+def test_locks(prog):
+    lock.scan_task(prog, 1)
```

Comment:
In Python, True and False are not integers. While this does work, it's more correct (and less confusing) to use the names:
```suggestion
    lock.scan_task(prog, True)
```

In fact, when you've renamed `stk` to `stack`, you can even use a keyword argument so it becomes even clearer:
```suggestion
    lock.scan_task(prog, stack=True)
```

Reply:
Done

Reply:
Done